src/pages/patientView/trialMatch/TrialMatchTableUtils.spec.tsx

```diff
@@ -0,0 +1,294 @@
+import React from 'react';
```

Comment:
spec files should be spec.ts (instead of tsx) if they do not use any JSX markup (e.g. <div>whatever</div>).

Reply:
Thanks for pointing it out. Will change to ts.